app/Libraries/Search/BeatmapsetSearch.php

```diff
@@ -48,12 +48,14 @@ public function __construct(?BeatmapsetSearchParams $params = null)
      */
     public function getQuery()
     {
+        static $partialMatchFields = ['artist', 'artist.*', 'artist_unicode', 'creator', 'description^0.5', 'title', 'title.raw', 'title.*', 'title_unicode', 'tags^0.5'];
```

Comment:
missing at least difficulty names (and maybe other fields?).

Reply:
@notbakaneko we gonna add difficulty here? probably a good idea (`version`)

Reply:
it's already handled by the other part of the query_string matcher; `$partialMatchFields` limits the fields combined in elasticsearch's query_string cross_field when doing partial terms matching